changelogs/fragments/8765-redfish-remove-required-image_url-for-virtual-media.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - redfish_utils - Make `image_url` an optional parameter for virtual media eject (https://github.com/ansible-collections/community.general/issues/3042, https://github.com/ansible-collections/community.general/pull/8765).
```

```suggestion
  - redfish_utils module utils - make ``image_url`` an optional parameter for virtual media eject (https://github.com/ansible-collections/community.general/issues/3042, https://github.com/ansible-collections/community.general/pull/8765).
```
Also please note that as a user this message is not helpful. Users cannot directly access the module util, they use modules or plugins of this collection. So you should better reference the module(s) this applies to.

---

Updated